src/common-utils/main.sh

```diff
@@ -311,6 +311,53 @@ install_alpine_packages() {
     PACKAGES_ALREADY_INSTALLED="true"
 }
 
+# Arch Linux packages
+install_arch_packages() {
```

Comment:
Can we add test scenarios to validate these PR changes? Thanks! 

https://github.com/devcontainers/features/blob/fff9a95f2242a79c83504bd9d4a56ad4b2fa2868/test/common-utils/scenarios.json